core/rawdb/schema.go

```diff
@@ -100,12 +100,14 @@ var (
 	CodePrefix            = []byte("c") // CodePrefix + code hash -> account code
 	skeletonHeaderPrefix  = []byte("S") // skeletonHeaderPrefix + num (uint64 big endian) -> header
 
+	// Path-based trie node scheme.
+	trieNodeAccountPrefix = []byte("A") // trieNodeAccountPrefix + hexPath -> trie node
+	trieNodeStoragePrefix = []byte("O") // trieNodeStoragePrefix + accountHash + hexPath -> trie node
```

Comment:
These two are correspond to `SnapshotAccountPrefix` and  `SnapshotStoragePrefix `. And prefix `A` and `O` are not occupied by any other data, so they are picked.